frontend/src/pages/CourseEditor/CourseEditorContainer.tsx

```diff
@@ -28,13 +30,17 @@ const CourseEditorContainer: FC = () => {
   const { enqueueSnackbar } = useSnackbar();
   const courseEditorRef = useRef<HTMLElement>(null);
   const [skillsById, setSkillsById] = useState<ISkillsById>({});
+  const [isSubmitButton, setSubmitButton] = useState(false);
+  const [showPrompt, confirmNavigation, cancelNavigation] = useCallbackPrompt(!isSubmitButton);
+  const [isLeavePageDialogOpen, setLeavePageDialogOpen] = useState(false);
```

Comment:
Should the modal window be implemented for both Course Editor and Course Creator in this task? @k-krasilnikova